docs/OpenCLIExamples.rst

```diff
@@ -0,0 +1,346 @@
+==========================
+CLI Open Endpoint Examples
+==========================
+
+The HCA CLI provides users of the Human Cell Atlas (HCA) to access and download data sets from the HCA. This page
```

Comment:
`Human Cell Atlas (HCA) to access and download data`

-> 

`Human Cell Atlas (HCA) access to download data`

Reply:
Fixed